src/ctia/http/routes/indicator.clj

```diff
@@ -35,26 +35,44 @@
     :tags ["Indicator"]
     (GET "/:id/judgements" []
       :return [StoredJudgement]
-      :path-params [id :- Long]
+      :path-params [id :- s/Str]
+      :query [params SightingsByIndicatorQueryParams]
       :summary "Gets all Judgements associated with the Indicator"
-      (not-found))
+      (if-let [indicator (read-indicator @indicator-store id)]
+        (if-let [judgements (list-judgements-by-indicators @judgement-store [indicator] params)]
```

Comment:
I'm trying to get it working locally.  I added this to ctia/http/routes/indicator.clj:

```clojure
(defn ->id [indicator-id]
  (id/->id :indicator
           indicator-id
           (get-in @properties [:ctia :http :show])))
```

Its a function that turns "short IDs" into CtiaID instances.